data/itsdevdeepak.json

```
@@ -1,19 +1,41 @@
 {
   "name": "Deepak",
-  "bio": "Fullstack Developer | Javascript/Typescript | Open Source",
+  "bio": "Web Alchemist 🧙‍♂️ | Crafting Digital Elixirs with JavaScript  🪄 | Exploring the Open Source Realm 💫",
+  "tags": ["Open Source", "Javascript", "Typescript", "Reactjs", "NextJs", "Nodejs"]
+  "socials": [
+    {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/itsdevdeepak"
+    },
+    {
+      "icon": "FaGithub",
+      "url": "https://github.com/itsdevdeepak"
+    },
+    {
+      "icon": "FaInstagram",
+      "url": "https://www.instagram.com/itsdevdeepak"
+    },
+    {
+      "icon": "FaLink",
+      "url": "https://itsdevdeepak.me"
+    },
+    {
+      "icon": "FaDollarSign",
+      "url": "https://github.com/sponsors/itsdevdeepak"
+    }
```

Also the missing comma and bracket here. 

```suggestion
    }],
```

---

done